client/src/Shared/Components/StyledComponents.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+import TrelloLogo from './TrelloLogo';
+import { color } from '../Utils/SharedStyles';
+
+export const StyledIcon = styled.i`
+  color: ${props => props.color};
+`;
+
+export const StyledTrelloLogo = styled(TrelloLogo)`
+  margin: ${props => props.margin ? props.margin : '0'};
+  &:hover {
```

Comment:
I don't think this actually works. Should figure out how to get the css selection working properly